pandas/core/generic.py

```diff
@@ -9244,7 +9246,7 @@ def _tz_convert(ax, tz):
                 raise ValueError(f"The level {level} is not valid")
             ax = _tz_convert(ax, tz)
 
-        result = self._constructor(self._data, copy=copy)
```

Comment:
self is a DataFrame or Series at this point, I suppose, and those still use `copy` in their init. So this change *might* have an impact?

Reply:
I don't think worth doing this without deprecating the `copy` argument to tz_convert, which is probably better as its own change